External/FEXCore/Source/Interface/Core/LookupCache.h

```diff
@@ -243,12 +243,12 @@ class LookupCache {
   // walking each block member and destructing objects.
   //
   // This makes `BlockLinks` look like a raw pointer that could memory leak, but since it is backed by the MBR, it won't.
-  std::pmr::monotonic_buffer_resource BlockLinks_mbr;
+  fextl::unique_ptr<std::pmr::monotonic_buffer_resource> BlockLinks_mbr;
```

Comment:
`unique_ptr` isn't really needed here - we can move initialization of `BlockLinks_mbr` into the constructor's initialization list instead (or initialize it right here in the header, but seems like a leaky implementation detail).